Assets/MixedReality.Toolkit.Foundation.nuspec

```diff
@@ -34,21 +34,30 @@
     <file src="..\Plugins\**\Microsoft.MixedReality.Toolkit.dll*" target="Plugins\" />
     <file src="..\Plugins\**\Microsoft.MixedReality.Toolkit.pdb*" target="Plugins\" />
     <file src="..\Plugins\**\Microsoft.MixedReality.Toolkit.Gltf.*" target="Plugins\" />
-    <file src="..\..\Assets\MixedRealityToolkit\**\*.cs" target="contentFiles\any\any\.PkgSrc\MRTK\MixedRealityToolkit" />
+    
+    <!-- The following line is commented out because of a path length issue in nuget for unity -->
```

Comment:
consider simply "because of a long path issue"

Reply:
we're not including the source files in the nuget package anymore?

Reply:
We haven't included it in any of the main releases so far:

2.1: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6251
2.2: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6618